components/log-viewer-webui/client/src/components/QueryBox/index.module.css

```
@@ -0,0 +1,27 @@
+.queryBox {
+  width: 100%;
+}
+
+.queryBoxPosition {
+  /* Enables position: absolute in Wrapper and Progress. */
```

if i understand correctly --
```suggestion
  /* Defines a new positioning context for absolutely positioned descendants such as the wrapper and progress bar */
```

(please reflow if we're violating 100-character line limit.